encrypt.go

```diff
@@ -505,6 +507,16 @@ func (e *imageEncrypter) run(opts *encryptOptions) error {
 	if err := e.setupWorkingDir(); err != nil {
 		return err
 	}
+	rootPartitionUUIDOverride := opts.RootPartitionUUID != ""
+	espPartitionUUIDOverride := opts.ESPPartitionUUID != ""
+
+	if rootPartitionUUIDOverride != espPartitionUUIDOverride {
+		return errors.New("Please override both root and efi partititions")
```

Comment:
The language should suggest that both overridden and neither overridden are the two acceptable states; the current wording only captures the former.